app/assets/appfilter.xml

```diff
@@ -779,7 +780,6 @@
   <item component="ComponentInfo{com.android.contacts/com.android.contacts.ContactsMainFromHomeActivity}" drawable="google_contacts" name="Contacts" />
   <item component="ComponentInfo{com.android.contacts/com.android.contacts.DialerActivity}" drawable="google_contacts" name="Contacts" />
   <item component="ComponentInfo{com.android.contacts/com.android.contacts.dialer.DialerPhoneActivity}" drawable="google_contacts" name="Contacts" />
-  <item component="ComponentInfo{com.android.contacts/com.android.contacts.DialtactsActivityAlias}" drawable="google_contacts" name="Contacts" />
```

Comment:
Why is this contact component link being removed?

Reply:
It wasn't removed. It was relinked as Phone. Because it was showing Contact icon instead of  the dialer icon.